src/utils/leaflet.js

```diff
@@ -0,0 +1,30 @@
+export const calculateMapBounds = points => {
+  let latitudes;
+  let longitudes;
+
+  const padding = 0.006;
+
+  if (!points.length) {
+    return null;
+  }
```

Comment:
This function may return different data types `Array` or `null` - this isn't usually considered as a best-practice. Intuitively I'd expect it to return an array, or throw an error that you'd handle above.

But I didn't dig deep into logic so this is an optional one. )

Reply:
Changed this func to return [ ] in case there is nothing provided.